tre_schemas/avro/dri-preingest-sip-available.avsc

```diff
@@ -1,60 +1,12 @@
 {
-  "namespace": "tna.tre.messages.drisip.available",
+  "namespace": "uk.gov.tna.tre.messages.drisip.available",
```

Comment:
If we're fully namespacing, should we use `nationalarchives` not `tna`(or is `tna` already reserved for this sort of purpose?)